.travis.yml

```diff
@@ -3,7 +3,7 @@ language: generic
 matrix:
   include:
     - os: linux
-      dist: trusty
+      dist: vivid
```

Comment:
That's not how that works.. https://docs.travis-ci.com/user/reference/overview/

Reply:
Ok :sweat_smile:  let me check